### ISSUE TITLE
Using backpack with blessed - preventing echoing input

### ISSUE THREAD

**Issue by hawkins** [palmerhq/backpack#22]
When running `backpack` as in `npm run dev`, input to the terminal is echoed back (type 'c' in your terminal and you immediately see 'c' in the terminal, like you'd generally expect).

However, [blessed](https://github.com/chjj/blessed) projects are expected to handle all screen actions themselves. When using backpack with a blessed project and running `backpack`, this input echoing messes up blessed's screen printing.

Example showing input echoed from moving mouse over terminal:

![image](https://cloud.githubusercontent.com/assets/9123458/21973190/889c17e4-db84-11e6-837c-56decff4545e.png)

This issue does not occur when running code built by `backpack build`, so I'm confident this could be remedied by disabling input echoing when running `backpack`. Is there a CLI flag to do this, or could you suggest how I might go about implementing this in either my script or as a PR for backpack?

**Comment by jaredpalmer** [comment]
Would something like `--silent=true` work?

**Comment by hawkins** [comment]
> Would something like `--silent=true` work?

Sure, that would work fine. Or just `--silent` and have it be false by default, true if included.

**Comment by jaredpalmer** [comment]
@hawkins would you be able to make a sample project on github to recreate the error? I am not familiar with `blessed`. 

I want to help as much as possible as simplifying writing CLI tools is one Backpack's strongest use-cases.

**Comment by hawkins** [comment]
Sure, I've created an example repository at [here](https://github.com/hawkins/backpack-blessed-example). 

The project shows logging to the screen constantly and, if you run `npm run backpack`, you'll see the issue occurs when you type any key in your terminal window. The keys you type should not be echoed back in a blessed program, since this ruins blessed's rendering. Running `npm run blessed` reveals the expected behaviour where keys are not echoed back onto the screen.

**Comment by jaredpalmer** [comment]
I think we need to figure out whether this is a nodemon, webpack watch mode, or related to webpack-friendly-errors. I will run some tests and report back.

**Comment by jaredpalmer** [comment]
So it's an issue with nodemon and blessed unfortunately. 

You can test this by running just `nodemon ./src/index.js` on your example project. Same issue happens with that and Backpack.  It looks like it was fixed temporarily and then reverted. 

See https://github.com/remy/nodemon/issues/890

**Comment by hawkins** [comment]
Blast, like you're right. 😞 I tried re-implementing their original 'fix', but then I discovered reloading blessed code was also error prone on scripts that affect blessed's screen.

Looks like for backpack to play well with bllessed would require disabling hot reloading of any scripts that render blessed's screen in addition to replacing nodemon, or patching both the 'inherit' fix on stdin, and it looks like there has not been any progress on since the issue you linked.

Since this is not backpack's issue, I'm closing this issue for now. Thanks for your help in troubleshooting this though!
